### ISSUE TITLE
Package not compatible with latest versions of fastapi

### ISSUE THREAD

**Issue by barapa** [JasperSui/fastapi-injectable#16]
Error when trying to install

```
Using version ^0.1.0 for fastapi-injectable

Updating dependencies
Resolving dependencies... (0.2s)

Because no versions of fastapi-injectable match >0.1.0,<0.2.0
 and fastapi-injectable (0.1.0) depends on fastapi (>=0.112.4,<0.113.0), fastapi-injectable (>=0.1.0,<0.2.0) requires fastapi (>=0.112.4,<0.113.0).
And because fastapi[all] (0.115.6) depends on fastapi (0.115.6)
 and no versions of fastapi match >0.115.6,<0.116.0, fastapi-injectable (>=0.1.0,<0.2.0) is incompatible with fastapi[all] (>=0.115.6,<0.116.0).
So, because legaide depends on both fastapi[all] (^0.115.6) and fastapi-injectable (^0.1.0), version solving failed.
```

**Comment by JasperSui** [comment]
Should be fixed by `0.1.1`

**Comment by barapa** [comment]
Confirmed - this fixed the issue
